bin/impl/load-env.sh

```diff
@@ -139,7 +139,7 @@ if [[ -z "$ZOOKEEPER_HASH" ]]; then
   echo "ZOOKEEPER_HASH is not set. Set it for your version in 'conf/checksums' or uno.conf"
   exit 1
 fi
-if [[ -z "$ACCUMULO_HASH" ]]; then
+if [[ -z "$ACCUMULO_HASH" && "$ACCUMULO_VERSION" != *"SNAPSHOT"* ]]; then
```

Comment:
I believe you don't need the double-quotes around $ACCUMULO_VERSION. On a similar note, I believe the double-quotes around SNAPSHOT are not needed as there are no embedded spaces in the "needle" string we are searching for. It is my understanding that the quotes would only be needed if there were embedded space(s) within the needle string.

Reply:
Thanks for the review @arvindshmicrosoft . Yes, right. I have included quotes mainly for consistency with the existing script

Reply:
Fair enough.